test/runtests.jl

```diff
@@ -31,7 +31,8 @@ projected = [
   Albers{23.0°,29.5°,45.0°},
   Sinusoidal,
   LambertAzimuthalEqualArea{15.0°},
-  EqualEarth
+  EqualEarth,
+  PolarStereographicB{-71°,0°,WGS84Latest}
```

Comment:
Notice that all CRS types omit the datum by default. The conversion rules and constructors take care of adding the default datum. Could you please review this to match the other CRS types?